docs/DeveloperGuide.md

```diff
@@ -283,32 +283,422 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 ### Use cases
 
-(For all use cases below, the **System** is the `AddressBook` and the **Actor** is the `user`, unless specified otherwise)
+(For all use cases below, the **System** is the `Healthcare Xpress` and the **Actor** is the `medical administrator`, unless specified otherwise)
 
-**Use case: Delete a person**
+**Use case: UC01 - Delete a patient/nurse**
 
 **MSS**
 
-1.  User requests to list persons
-2.  AddressBook shows a list of persons
-3.  User requests to delete a specific person in the list
-4.  AddressBook deletes the person
+1.  Medical administrator requests to list patients,nurses or both according to general or specific groups.
+2.  Healthcare Xpress shows a list of patients/nurses that satisfy the group's condition.
+3.  Medical administrator requests to delete a specific patient/nurse in the list.
+4.  Healthcare Xpress deletes the patient/nurse.
 
     Use case ends.
 
 **Extensions**
+* 1a. The given input is invalid.
 
+    * 1a1. Healthcare Xpress shows an error message.
+
+      Use case restart. 
+
+* 1b. Medical administrator get a specific patient/nurse.
```

Comment:
I am wondering if it is possible to omit this, because it seems like the number of steps is still the same, the only difference is that for 1b1, healthcare xpress displays only 1 record whereas for step 2, healthcare xpress displays >1 record. Since there is no difference in the number of steps, I don't think we need to handle this case differently. What are your thoughts?

Reply:
> Hmm I also not sure about it cuz like I wrote that to show that not only list but can also use get n delete that specific person. But it is repetitive , so hmmm I also not very sure.

Okay maybe we leave it like that for now.

> Btw, one question, the textbook didnt say anything ah but can like add extensions for extensions (like the method is in extensions already but if do that method, it can lead to another extensions sth like that)

Yeah should be can but try not to recurse too much haha